docs/Synthesis UI.md

```diff
@@ -0,0 +1,92 @@
+# UI interactions in various modes
+
+Document describes modes that browser.html could be in & how various UI components will behave in that mode based of the **Synthesis UI / Jun 15, 2015** specification.
+
+## Actions
+
+- `ShowWebView`
+- `ShowWebViewByID`
+- `CreateWebView`
+- `EditWebView`
+- `ChooseNextWebView`
+- `ChoosePreviousWebView`
+- `Escape`
+- `CloseWebView`
+
+## Modes
+
+Application state will have a `mode` of string type that maybe set to on of the following values:
+
+- `'show-web-view'`
```

Comment:
We will also need a way to define animation transition for navigating `webview -> dashboard` and `dashboard -> webview`. These are different depending on context. So when clicking `+` we `webview -> dashboard` with a zoom-like animation. But when `alt tab` we `webview -> dashboard` with no animation at all (see spec).